blog/2019-12-18-HowToPlanAndOptimizeASecretSanta.adoc

```diff
@@ -0,0 +1,102 @@
+= How to Plan (and Optimize) a Secret Santa
+:page-interpolate: true
+:awestruct-author: cchianel
+:awestruct-layout: blogPostBase
+:awestruct-tags: [useCase]
+
+Many businesses host a "Secret Santa", where each employee gets assigned
+a coworker whom they need to buy a present for. This fosters good relations
```

Comment:
```suggestion
a coworker whom they ought to buy a present for. This fosters good relations
```
Maybe it's just me not being a native speaker, but "need" seems too direct for something fun that you do voluntarily.

Reply:
Well, if you volunteer, you cannot refuse giving a present for the person you were assigned (without asking the organizer, and then needing to do non-disruptive planning, which for Secret Santa is very hard).